keps/sig-windows/2258-node-service-log-viewer/README.md

```diff
@@ -124,16 +128,16 @@ opportunities for tooling and simplifying automated log gathering. The feature
 can also be used to debug issues with Kubernetes services especially in Windows
 nodes that run as native Windows services and not as DaemonSets or Deployments.
 
-Here are some example of how a cluser administrator would use this feature:
+Here are some example of how a cluster administrator would use this feature:
 ```
 # Show kubelet and crio journal logs from all masters
-kubectl logs nodes --role master -s kubelet -s crio
+kubectl logs nodes --role master -q kubelet -q crio
```

Comment:
Maybe I missed this before, but this seems like a bad overload of the `logs` command.  If I have a pod named `nodes` (which is weird but valid) then `kubectl logs nodes` is ambiguous.

I think this needs a different top-level command, like `kubectl nodelogs` - what do you think?

Reply:
Sure, that works for me, though I think `node-logs` reads better.

Reply:
I'm fine with node-logs - someone from sig CLI shoul dget final say on that :)